lib/queue/worker.py

```diff
@@ -1,10 +1,13 @@
+import os
+from concurrent.futures import ThreadPoolExecutor, TimeoutError
 import json
-from typing import List
+from typing import List, Tuple
 from lib import schemas
 from lib.logger import logger
 from lib.queue.queue import Queue
 from lib.model.model import Model
 from lib.helpers import get_setting
+TIMEOUT = int(os.getenv("WORK_TIMEOUT", "60"))
```

Comment:
add comment to indicate units?  (seconds?)

Reply:
Fair enough!